Parse/Internal/Commands/CommandRunner/URLSession/PFURLSessionCommandRunner.m

```diff
@@ -49,11 +55,12 @@ - (instancetype)initWithDataSource:(id<PFInstallationIdentifierStoreProvider>)da
                          clientKey:(NSString *)clientKey {
     NSURLSessionConfiguration *configuration = [[self class] _urlSessionConfigurationForApplicationId:applicationId
                                                                                             clientKey:clientKey];
-    PFURLSession *session = [PFURLSession sessionWithConfiguration:configuration];
+    PFURLSession *session = [PFURLSession sessionWithConfiguration:configuration delegate:self];
```

Comment:
Sure, the possibility of that happening is really really close to 0, so better testability (with injected everything) vs rare edge case scenario.